### DISCUSSION TITLE
Values with / represents multiple types of links

### DISCUSSION THREAD

**Discussion by avvalex** [Koenkk/zigbee2mqtt#6765] [General]
In the frontend map it's explained that "values with / represents multiple types of links"... But what does that means exactly?
For example, the Sonoff ZBMINI "Luce Alex" has a LQI of 36, and apparently seems connected only with the coordinator. The link quality measurement is always around 28-39 (it lowers when the switch is on, maybe caused by EMI), nothing strange so far.
But i've noticed that also the other number changes, between 172 and 178. Where this number il taken from? Maybe one is the LQI from Coordinator to Sonoff and the other is the LQI from Sonoff to Coordinator?
I am new to Zigbee world, maybe this is a stupid question, sorry.....

<img width="1152" alt="Immagine" src="https://user-images.githubusercontent.com/60694870/111841802-d1ed6980-88fe-11eb-90e4-401ea04947ea.png">

**Comment by kbfifi** [comment]
I hope this question gets answered as I have the same question
